src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img class="logo" src="logo.png" alt="logo">
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item" >
+            <a class="link apple is-active" href="#">
+              Apple
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#">
+              Samsung
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="link" href="#">
+              Smartphones
+            </a>
+          </li>
+          <li class="nav__item">
+            <a class="link" data-qa="hover" href="#">
+              Laptops &amp; computers
+            </a>
```

Comment:
whenever you have more than 2 attributes it's better to write them from a new line
```suggestion
            <a 
              class="link" 
              data-qa="hover" 
              href="#"
            >
              Laptops &amp; computers
            </a>
```